packages/blade-old/src/atoms/Badge/Badge.native.js

```diff
@@ -38,9 +38,9 @@ const styles = {
         return getColor(theme, 'primary.900');
       default:
         if (fill === 'subtle') {
-          return getColor(theme, 'positive.930');
+          return variant ? getColor(theme, variant) : getColor(theme, 'positive.930');
```

Comment:
Once we start passing colors as variant, how will we make sure people are not passing colors which are not in theme, also does getColor function has a default case color ?

Reply:
getColor throws error on wrong color name